src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,113 @@
+| [dart:convert](https://api.dartlang.org/stable/dart-convert/dart-convert-library.html)<br>Encoders and decoders for converting between different data representations, including JSON and UTF-8.| [convert](https://www.dartdocs.org/documentation/convert/latest/)<br>This package contains encoders and decoders for converting between different data representations.|
+| [dart:core](https://api.dartlang.org/stable/dart-core/dart-core-library.html)<br>Built-in types, collections, and other core functionality for every Dart program. | N/A |
+| dart:js<br>Deprecated. | [js](https://www.dartdocs.org/documentation/js/latest)<br>Methods and annotations to specify interoperability with JavaScript APIs. Use this package instead of dart:js. |
+{:.table .table-striped .nowrap}
+
+## Other useful packages
+
+Developers have written some excellent packages for use by the
+Dart community.  Other popular and useful packages that we recommend,
+in alphabetical order:
+
+[firebase3](https://pub.dartlang.org/packages/firebase3)
+: A Dart wrapper library for the Firebase.
+
+[intl](https://pub.dartlang.org/packages/intl)
+: Support for internationalization and localization facilities.
+
+[logging](https://pub.dartlang.org/packages/logging)
+: A configuratable mechanism for logging messages.
+
+[mockito](https://pub.dartlang.org/packages/mockito)
+: A Dart version of the Mockito framework.
+
+[path](https://pub.dartlang.org/packages/path)
+: A comprehensive, cross-platform path manipulation library.
+  See [Unboxing Packages: path](http://news.dartlang.org/2016/06/unboxing-packages-path.html)
+  for more information.
+
+[quiver](https://pub.dartlang.org/packages/quiver)
+: A set of utilitiy libraries that makes it easier and more convenient to use many Dart
+  libraries.
+
+[shelf](https://pub.dartlang.org/packages/shelf)
+: Web server middleware for Dart.
+
+[stagehand](https://pub.dartlang.org/packages/stagehand)
+: A Dart project generator.
+
+[test](https://pub.dartlang.org/packages/test)
+: A standard way of writing and running tests in Dart.
+
+Search for other available packages on [pub.dartlang.org](https://pub.dartlang.org/).
+
+## "Unboxing Packages" series
+
+Natalie Weizenbaum has written a series of posts on
+[news.dartlang.org](http://news.dartlang.org/) called "Unboxing Packages."
+Each post contains in-depth detail on how to use some of the Dart packages:
+
+* [Unboxing Packages: async, part 1](http://news.dartlang.org/2016/03/unboxing-packages-async-part-1.html)
+* [Unboxing Packages: async, part 2](http://news.dartlang.org/2016/03/unboxing-packages-async-part-2.html)
+* [Unboxing Packages: async, part 3](http://news.dartlang.org/2016/04/unboxing-packages-async-part-3.html)
+* [Unboxing Packages: collection](http://news.dartlang.org/2016/01/unboxing-packages-collection.html)
+* [Unboxing Packages: json_rpc_2](http://news.dartlang.org/2016/05/unboxing-packages-jsonrpc2.html)
```

Comment:
It doesn't seem helpful to link to a post about a library that we don't discuss on this page.